### DISCUSSION TITLE
What is the reason why the lyrics cannot be loaded when loading the json file? The console prints have lyrics

### DISCUSSION THREAD

**Discussion by hhly** [CoderLine/alphaTab#1021] [Q&A] [answered]
What is the reason why the lyrics cannot be loaded when loading the json file? The console prints have lyrics
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/29226405/198015671-4197921f-ab17-428c-b143-9b19a765fab2.png">

**Comment by Danielku15** [comment, marked as answer]
Please share some more insights on your application and code. From a screenshot nobody will be able to help here.

**Reply by hhly** [reply]
This problem has been solved, thank you
